src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/src/App.test.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+    const storeFake = (state: any) => ({
+        default: () => {},
+        subscribe: () => {},
+        dispatch: () => {},
+        getState: () => ({ ...state })
+    });
+    const store = storeFake({});
+
+    ReactDOM.render(
+        <Provider store={store}>
+            <MemoryRouter>
+                <App/>
+            </MemoryRouter>
+        </Provider>, document.createElement('div'));
+});
```

Comment:
Minor point: Normally we end our files with a trailing newline. Some of the files in the PR do end with a newline, some don't. Might be good to make them all consistent.

Reply:
Fixed.